bench/rand.cc

```
@@ -0,0 +1,59 @@
+#include <openssl/evp.h>
+#include <openssl/mem.h>
+#include <openssl/rand.h>
+
+#include <benchmark/benchmark.h>
+
+#include "./internal.h"
+
+namespace {
+
+void BM_SpeedRandom(benchmark::State &state) {
+  size_t buf_size = static_cast<size_t>(state.range(0));
```

Clang (14.0.6)

[warning] variable 'buf_size' is not initialized

(cppcoreguidelines-init-variables)